files/scripts/sonic-netns-exec

```diff
@@ -6,7 +6,7 @@
 NS="$1"
 shift
 if [ "$NS" != "" ]; then
-    ip netns exec $NS $@
+    ip netns exec $NS "$@"
```

Comment:
this change does not look correct to me.

check below

```
lgh@gulv-vm3:/data2/sonic/stretch/sonic-buildimage$ sudo ip netns exec abc ip addr show
1: lo: <LOOPBACK> mtu 65536 qdisc noop state DOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
lgh@gulv-vm3:/data2/sonic/stretch/sonic-buildimage$ sudo ip netns exec abc "ip addr show"
exec of "ip addr show" failed: No such file or directory
lgh@gulv-vm3:/data2/sonic/stretch/sonic-buildimage$ 
```

Reply:
Right, but as we are adding quotes inside the sonic-netns-exec script, it will make sure that we are executing it as a single command. 
For example:
admin@NAMESPACE5:~$ sudo sonic-netns-exec asic0 ifconfig eth0
eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
..
admin@NAMESPACE5:~$ sudo sonic-netns-exec asic0 echo "hello" 
hello

Reply:
as my example, if you quote the entire command, then it does not work.  how do you explain that?

Reply:
When sonic-netns script is executed, the quotes are not added in the final command itself.